python/cudf/cudf/_lib/cpp/wrappers/decimals.pxd

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+from libc.stdint cimport int64_t, int32_t
+
+cdef extern from "cudf/fixed_point/fixed_point.hpp" namespace "numeric" nogil:
+    ctypedef int64_t decimal64
+
+    cdef cppclass scale_type:
+        scale_type(int32_t)
+        int32_t operator()() except +
```

Comment:
Is this wrapper used?

Reply:
I think we still use it to set the scalar in `_set_decimal64_from_scalar` - using the overload of `fixed_point_scalar` which requires a scale

Reply:
Sorry, I mean the `operator ()` wrapper?

Reply:
removed 👍